api/api.go

```diff
@@ -145,6 +145,13 @@ const (
 	apiV2VulnerabilityExceptionFromGUID = "v2/VulnerabilityExceptions/%s"
 
 	apiV2EntitiesSearch = "v2/Entities/%s/search"
+
+	apiV2Alerts       = "v2/Alerts"
+	apiV2AlertsByTime = "v2/Alerts?startTime=%s&endTime=%s"
+	//apiV2AlertsSearch  = "v2/Alerts/search"
```

Comment:
Are you implementing this? or you are leaving it for later?

Reply:
@afiune , this is described in the `What's not included section above` and is being tracked as:
Alert search (https://lacework.atlassian.net/browse/ALLY-1206)